readme.md

```diff
@@ -436,6 +436,7 @@
 - [Educational Games](https://github.com/yrgo/awesome-eg#readme) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning#readme)
 - [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.
+- [Product Management](https://github.com/dend/awesome-product-management) - Resources for Product Managers: tools, articles, books, podcasts and more.
```

Comment:
```suggestion
- [Product Management](https://github.com/dend/awesome-product-management#readme) - Resources for Product Managers: tools, articles, books, podcasts and more.
```

Reply:
@sindresorhus addressed this as well - thanks again for looking over the PR!